middlewares/authenticate.js

```diff
@@ -34,7 +34,7 @@ module.exports = async (req, res, next) => {
     const decoded = authService.verifyAuthToken(token)
 
     // add user data to `req.userData` for further use
-    const userData = await users.fetchUser(decoded.userId)
+    const userData = await users.fetchUser(null, decoded.userId)
```

Comment:
**Suggestion: **
Passing `null` as an argument does not feel good.
Is it a better idea to make the argument as an Object which will contain `userId` and `userName`

The function call will look like this
```
const userData = await users.fetchUser({
    userId: 21389374,
    userName: 'Swarajpure'
})
```

Opinions: @ankurnarkhede @swarajpure

Reply:
Awesome idea @whyDontI !
Had not at all thought of it.
Pushed commit with this change. Please review.

Thank you so much! 😄